Dockerfile

```diff
@@ -0,0 +1,11 @@
+# latest base image without vulnerabilities
+FROM python:3.5.7-alpine3.10
```

Comment:
Any reason we don't use Python 3.7 e.g. `3.7.4-alpine3.10` ? Another option (what I would prefer) is using [distroless](https://github.com/GoogleContainerTools/distroless/issues/171).

Reply:
according to https://hub.docker.com/_/python/scans/library/python/3.7-alpine3.10 this image has several vulnerabilities,
at the end it does not really matter which image is used to I took the first image without known vulnerabilities

Reply:
I get a 404 🤔 are these scans scheduled or only done once when pushing it ?

Reply:
Oh I need to login to see the scan.. wtf Docker.. Both images have the exact same CVE's so I see no benefit in using an older Python version. I would vote for 3.7 (since this is the version our e2e also runs on).